src/iter/mod.rs

```diff
@@ -68,3 +68,29 @@ impl<Pk: MiniscriptKey, Ctx: ScriptContext> TreeLike for Arc<Miniscript<Pk, Ctx>
         }
     }
 }
+
+impl<'a, Pk: MiniscriptKey> TreeLike for &'a policy::Concrete<Pk> {
```

Comment:
This is really not a concern for this PR, but I think policy stuff should belong in policy module. Not here. See #603 for more.